springwolf-core/src/main/java/io/github/stavshamir/springwolf/schemas/example/ExampleJsonValueGenerator.java

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.stavshamir.springwolf.schemas.example;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.BooleanNode;
+import com.fasterxml.jackson.databind.node.DoubleNode;
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.swagger.v3.core.util.Json;
+import jakarta.validation.constraints.NotNull;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+
+@Slf4j
+public class ExampleJsonValueGenerator implements ExampleValueGenerator<JsonNode, JsonNode> {
+
+    private static final BooleanNode DEFAULT_BOOLEAN_EXAMPLE = BooleanNode.TRUE;
+
+    private static final ObjectMapper objectMapper = Json.mapper();
+    private static final Double DEFAULT_NUMBER_EXAMPLE = 1.1;
+    private static final Integer DEFAULT_INTEGER_EXAMPLE = 0;
+
+    private static final String DEFAULT_DATE_EXAMPLE = "2015-07-20";
+    private static final String DEFAULT_DATE_TIME_EXAMPLE = "2015-07-20T15:49:04-07:00";
+    private static final String DEFAULT_PASSWORD_EXAMPLE = "string-password";
+    private static final String DEFAULT_BYTE_EXAMPLE = "YmFzZTY0LWV4YW1wbGU=";
+    private static final String DEFAULT_BINARY_EXAMPLE =
+            "0111010001100101011100110111010000101101011000100110100101101110011000010110010001111001";
+    private static final String DEFAULT_STRING_EXAMPLE = "string";
+    private static final String DEFAULT_EMAIL_EXAMPLE = "example@example.com";
+    private static final String DEFAULT_UUID_EXAMPLE = "3fa85f64-5717-4562-b3fc-2c963f66afa6";
+
+    private static String DEFAULT_UNKNOWN_SCHEMA_EXAMPLE(String type) {
+        return "unknown schema type: " + type;
+    }
+
+    private static String DEFAULT_UNKNOWN_SCHEMA_STRING_EXAMPLE(String format) {
+        return "unknown string schema format: " + format;
+    }
+
+    @Override
+    public boolean canHandle(String contentType) {
+        return (StringUtils.equals(contentType, "application/json"));
```

Comment:
What about yaml?

Should be fallback to json (I guess this is given implicitly)?